tests/NexusMods.Collections.Tests/CollectionInstallTests.CanInstallBasicCollection.verified.txt

```diff
@@ -2,7 +2,7 @@
   mods: [
     Game Files,
     Halgari's Helper,
-    My Collection
```

Comment:
How is this not breaking the tests before or after?

Reply:
It's a carryover from last PR, it was broken before in that case. Truth be told I'm curious myself.